### PR TITLE
Public Cloud: Enable CLI Tools on 12-SP5, 15-SP1 and 15-SP2

### PR DESCRIPTION
**Pull request by pdostal**
- Related Job Groups merge request: [qac-openqa-yaml!640](https://gitlab.suse.de/qac/qac-openqa-yaml/-/merge_requests/640)
- Related metadata merge request: [metadata!580](https://gitlab.suse.de/qa-maintenance/metadata/-/merge_requests/580)
- Related ticket: [poo#106194](https://progress.opensuse.org/issues/106194) [bsc#1195351](https://bugzilla.suse.com/show_bug.cgi?id=1195351) [bsc#1192671](https://bugzilla.suse.com/show_bug.cgi?id=1192671)
- Verification run: [12-SP5](http://pdostal-server.suse.cz/tests/13340) [15-SP1](http://pdostal-server.suse.cz/tests/13339), [15-SP2](http://pdostal-server.suse.cz/tests/13341), [15-SP3](http://pdostal-server.suse.cz/tests/13327)

### CODE REVIEW

File: tests/publiccloud/aws_cli.pm [hunk 1]
```diff
@@ -24,7 +25,8 @@ sub run {
 
     # If 'aws' is preinstalled, we test that version
     if (script_run("which aws") != 0) {
-        add_suseconnect_product 'sle-module-public-cloud';
+        add_suseconnect_product(get_addon_fullname('pcm'), (is_sle('=12-sp5') ? '12' : undef));
+        add_suseconnect_product(get_addon_fullname('phub')) if is_sle('=12-sp5');
```
Comment (grisu48) [inline tests/publiccloud/aws_cli.pm:29]: Why is PackageHub added here?

File: tests/publiccloud/azure_cli.pm [hunk 2]
```diff
@@ -24,7 +25,8 @@ sub run {
 
     # If 'az' is preinstalled, we test that version
     if (script_run("which az") != 0) {
-        add_suseconnect_product 'sle-module-public-cloud';
+        add_suseconnect_product(get_addon_fullname('pcm'), (is_sle('=12-sp5') ? '12' : undef));
+        add_suseconnect_product(get_addon_fullname('phub')) if is_sle('=12-sp5');
```
Comment (grisu48) [inline tests/publiccloud/azure_cli.pm:29]: Why is PackageHub added here?

File: tests/publiccloud/aws_cli.pm (hunk 1 above)
Comment (pdostal) [reply tests/publiccloud/aws_cli.pm:29]: It is needed for `jq`.

File: tests/publiccloud/azure_cli.pm (hunk 2 above)
Comment (pdostal) [reply tests/publiccloud/azure_cli.pm:29]: It is needed for `jq`.

**Review by grisu48 — APPROVED** [review]

**Comment by jlausuch** [comment]
LGTM!

**Pull request merged**